versions.md

```diff
@@ -1,6 +1,8 @@
 Forthcoming
 -----------
 
+* New Fast DDS Statistics module (implies ABI break)
```

Comment:
I don't think this is necessarily true. Not on the DDS API at least

Reply:
But we are going to add new methods on the RTPS entities, so the ABI break will be there anyways

Reply:
I agree, I just won't write it here until they happen. This PR does not entail an ABI break, so I won't state it yet.

Reply:
The statement saying that `implies ABI break` has been removed in 19ab1b3d01118aff59fdfc5e958b494b2659a729